### PR TITLE
Load videos

### PR DESCRIPTION
**Pull request by bent0b0x**

### CODE REVIEW

**Comment by bent0b0x** [comment]
@eliawk if you have a sec can you merge `dev` into this branch? I think that would pass the build

**Comment by bent0b0x** [comment]
Actually not entirely true, unit tests need to be updated as well. But I think this needs `dev` merged into it anyway

**Comment by eliawk** [comment]
@bent0b0x i see the layout kind broken
schermata 2018-02-23 alle 12 57 30

the video list at the first upload it's gone away

**Comment by bent0b0x** [comment]
@eliawk are you sure this issue is not happening on `dev` too? I do not see how this would be related or caused by this PR.

File: src/scripts/records/VideoRecords.js [hunk 1]
```diff
@@ -16,6 +16,9 @@ class Video extends ImmutableRecord({
   price: '',
   thumbnailUrl: '',
   title: '',
+  author: '',
+  free: '',
```
Comment (bent0b0x) [inline src/scripts/records/VideoRecords.js]: I feel like we shouldn't have a `free` field. Doesn't that just mean `price === '0'`? This seems like a duplication of state and we should avoid

**Pull request merged**